app/controllers/comments_controller.rb

```diff
@@ -6,6 +6,10 @@ def create
       comment: params[:comment][:comment],
     )
 
+    unless edition.notification_subscribers == [comment.user]
+      NotificationMailer.comment_added(comment).deliver_later
```

Comment:
It's a really cool new Rails API, defaults to synchronous job execution but as soon as we add a queue processor it will do it async without any change to the code!